app/assets/stylesheets/application.scss

```diff
@@ -61,30 +70,24 @@ $indianred: #e45e56;
   text-align: center;
 }
 
-// Footer
-
-.page-wrap {
-	margin-bottom: -142px;
-  min-height: 100%;
-}
-
-.page-wrap:after {
-  content: "";
-  display: block;
-}
+$footer-height: 100px;
 
-.site-footer,
-.page-wrap:after {
-  height: 142px;
+.wrapper {
+  min-height: calc(100% - #{$footer-height});
+	background-image: image_url("turks.jpg");
```

Comment:
Line should be indented with spaces, not tabs<br>Name of function `image_url` should be written in all lowercase letters with hyphens instead of underscores